pkg/virt-config/config_test.go

```diff
@@ -51,6 +51,18 @@ var _ = Describe("ConfigMap", func() {
 		table.Entry("when invalid, it should return the default", "invalid", false),
 	)
 
+	table.DescribeTable(" when permitBridgeInterfaceOnPodNetwork", func(value string, result bool) {
+		clusterConfig, _ := testutils.NewFakeClusterConfig(&kubev1.ConfigMap{
+			Data: map[string]string{"PermitBridgeInterfaceOnPodNetwork": value},
+		})
+		Expect(clusterConfig.IsBridgeInterfaceOnPodNetworkEnabled()).To(Equal(result))
+	},
+		table.Entry("is true, it should return true", "true", true),
```

Comment:
(I understand that you copy-pasted but) what is "it" here? Whoever runs the tests will see a message similar to: "[ConfigMap] when permitBridgeInterfaceOnPodNetwork is true, it should return true" when an error happens. I think you should have the name of the function you are testing against somewhere embedded in the message.

Reply:
you mean something like : 
...("is true, IsBridgeInterfaceOnPodNetworkEnabled should return true","true", true)...
?

Reply:
You are not required to change it for existing tests; I comment on the new code only. I am fine if you do though.